src/test/java/org/openelisglobal/person/PersonServiceTest.java

```diff
@@ -22,9 +26,33 @@ public class PersonServiceTest {
 
     @Autowired
     PersonService personService;
+    private String firstName;
+    private String lastName;
+    private String personId;
+    private String cellPhone;
+    private String workPhone;
+    private String email;
+    private String fax;
+    private String streetAddress;
+    private String city;
+    private String country;
+    private String zip;
+    private String state;
 
     @Before
     public void init() throws Exception {
+        firstName = "Malik";
+        lastName = "Sardar";
+        personId = "1234";
+        cellPhone = "403-678-5678";
```

Comment:
Phone number should start with country code for the test to more clear. 

Some merge Conflicts to solve.